### PR TITLE
feat(cli): add get poll cli command

### PR DESCRIPTION
**Pull request by 0xmad**
# Description

- [x] Add get poll cli command
- [x] Add indexed params for deploy poll event

## Additional Notes

N/A

## Related issue(s)

Related to https://github.com/privacy-scaling-explorations/maci-rpgf/issues/10

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: cli/ts/commands/poll.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  if (pollAddress === ZeroAddress) {
+    logError("MACI contract doesn't have any deployed poll");
+  }
+
+  const pollContract = PollFactory.connect(pollAddress, signer);
+
+  const [[deployTime, duration], isStateAqMerged, numSignups] = await Promise.all([
+    pollContract.getDeployTimeAndDuration(),
+    pollContract.stateAqMerged(),
+    pollContract.numSignups(),
```
Comment (ctrlc03) [inline cli/ts/commands/poll.ts:36]: so the number of signups will only be known after the state aq is merged. what about taking it from maci instead which will have up to date info if the poll is not completed yet?

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Nice thank you! just two comments

File: cli/ts/commands/poll.ts (hunk 1 above)
Comment (ctrlc03) [reply cli/ts/commands/poll.ts:36]: or maybe to be even more precise, we first can check if the state aq are merged, if they are, take from poll. if not take from maci

Comment (0xmad) [reply cli/ts/commands/poll.ts:36]: Does it update when new poll is deployed? If no, I can just remove it.

Comment (ctrlc03) [reply cli/ts/commands/poll.ts:36]: on maci, numSignups can keep growing poll after poll as new users signup. whereas on a specific poll, we store the number of users signed up up until the poll is finalized

**Review by ctrlc03 — APPROVED** [review]
Thank you!!

**Pull request merged**